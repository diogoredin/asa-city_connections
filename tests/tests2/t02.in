100
100
62 82
2 4
11 7
4 34
44 14
66 59
39 61
19 79
46 59
32 82
75 70
91 28
93 87
34 99
27 22
74 28
77 53
63 24
1 46
42 37
21 25
22 68
23 70
13 25
24 58
57 86
8 60
83 51
72 28
31 1
3 85
40 63
90 56
98 49
94 91
49 35
73 61
86 6
33 72
52 93
9 55
30 70
16 99
18 40
35 3
41 1
47 69
48 39
6 57
99 91
84 56
12 32
36 97
17 49
80 68
56 13
58 18
51 30
96 74
61 52
55 43
82 55
45 13
92 98
43 86
15 61
5 71
70 89
10 29
65 64
37 52
7 30
68 51
38 77
88 56
67 89
87 84
81 1
79 30
76 70
54 32
50 10
100 41
20 74
85 59
69 37
29 26
59 89
89 98
53 73
14 25
64 58
71 73
60 6
95 5
78 44
26 82
97 100
25 37
28 41
100
35 14 83
44 23 49
39 15 29
24 36 1
8 78 85
47 12 29
4 42 15
4 31 25
74 9 28
87 85 8
91 18 77
39 59 14
19 34 41
39 33 19
99 26 60
73 17 53
69 59 2
10 37 57
1 81 88
91 72 5
51 7 82
84 58 69
5 78 67
77 48 83
71 28 22
6 96 38
32 72 89
75 65 25
8 72 100
39 20 72
80 28 80
82 38 14
58 67 80
32 50 37
2 94 62
44 18 30
45 10 91
3 49 79
88 85 37
9 99 66
93 20 91
93 79 49
63 26 71
72 43 96
66 70 96
27 28 40
63 65 44
62 29 20
72 30 30
13 26 37
46 10 44
8 52 23
55 92 97
43 89 59
76 89 22
3 10 41
42 37 88
66 73 21
77 43 98
15 29 25
66 18 64
68 53 22
68 54 60
91 25 41
16 15 9
23 40 12
84 60 36
62 60 94
56 64 20
9 70 91
24 96 63
57 35 70
66 14 43
56 76 75
61 95 82
55 95 100
91 24 23
92 28 60
34 33 5
73 86 45
31 90 65
46 74 92
57 68 87
75 100 43
1 13 33
16 87 46
19 21 56
11 1 29
83 18 47
30 84 38
2 90 35
7 38 31
1 41 52
75 98 6
22 56 8
40 38 34
68 28 55
44 97 40
41 47 38
22 12 76
