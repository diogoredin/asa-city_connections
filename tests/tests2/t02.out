3118
44 56
